qiita_pet/webserver.py

```diff
@@ -106,6 +108,11 @@ def __init__(self):
             (r"/stats/", StatsHandler),
             (r"/download/(.*)", DownloadHandler),
             (r"/vamps/(.*)", VAMPSHandler),
+            # Plugin handlers - the order matters here so do not change
+            (r"/qiita_db/jobs/(.*)/heartbeat/", HeartbeatHandler),
```

Comment:
Yes, the handlers are in a different location, but we decided in the last meeting that we're gonna be using the same webserver in interests of time.